packages/components/autocomplete/README.mdx

```diff
@@ -96,6 +96,14 @@ In order to use proper form validation you need to be able to control the actual
 
 ```
 
+### Custom icon
+
+Pass a custom icon to the text input
```

Comment:
Maybe we could give some recommendation when it might be useful. Just a thought 💭